mirdata/datasets/simac.py

```diff
@@ -0,0 +1,226 @@
+
+    - **Timbre and Instrumentation:** SIMAC focuses on the overall timbre or texture of music, as current technologies do not allow for reliable separation of individual instrumental information. This aspect is characterized based on low-level signal features.
+
+    - **Music Structure:** The project examines how music materials are presented, repeated, varied, or confronted in a piece, providing ways to interact with audio content through summaries, fast-listening, and on-the-fly identification of songs.
+
+    - **Intensity and Complexity:** These descriptors are defined to capture the subjective sensation of energeticness and the effort required to follow and understand a musical piece. The project notes the relationship between music complexity and listener preference.
+
+    - **Music Similarity:** SIMAC tackles the challenge of defining music similarity, considering both audio-based aspects and cultural background. It aims to develop similarity metrics that incorporate multiple musical facets beyond just timbre.
+
+    **Future Directions and Challenges:**
+
+    SIMAC identifies areas for future exploration, such as incorporating musical facets beyond timbre in similarity metrics and addressing the limitations of current approaches to similarity. The project underscores the potential benefits of enriching music files with metadata from their origin for more effective music retrieval and recommendation systems.
+
+    **Acknowledgments and References:**
+
+    The project involved more than 15 collaborators and was led by teams from Universitat Pompeu Fabra Barcelona, Queen Mary University London, the Austrian Research Institute for Artificial Intelligence Vienna, and Philips Research Eindhoven. For detailed information, visit [http://www.semanticaudio.org](http://www.semanticaudio.org).
+"""
+
+import os
+import csv
+import logging
+import librosa
+import numpy as np
+from typing import BinaryIO, Optional, TextIO, Tuple
+
+from mirdata import annotations, core, download_utils, io, jams_utils
+
+
+BIBTEX = """
+@INPROCEEDINGS{1576040,
+  author={Herrera, P. and Bello, J. and Widmer, G. and Sandler, M. and Celma, O. and Vignoli, F. and Pampalk, E. and Cano, P. and Pauws, S. and Serra, X.},
+  booktitle={The 2nd European Workshop on the Integration of Knowledge, Semantics and Digital Media Technology, 2005. EWIMT 2005. (Ref. No. 2005/11099)}, 
+  title={SIMAC: semantic interaction with music audio contents}, 
+  year={2005},
+  volume={},
+  number={},
+  pages={399-406},
+  keywords={},
+  doi={10.1049/ic.2005.0763}}
+"""
+
+INDEXES = {
+    "default": "1.0",
+    "test": "1.0",
+    "1.0": core.Index(filename="simac_full_index_1.0.json"),
+}
+
+REMOTES = None
+
+LICENSE_INFO = (
+    "Creative Commons Attribution Non Commercial Share Alike 4.0 International."
+)
+
+DOWNLOAD_INFO = """
+    Unfortunately most of the simac dataset is not available for download.
```

Comment:
As suggesed for #634, it'd be a bit more conscise on what `most of the` means here.